app/views/notification/AddGuidanceView.scala.html

```diff
@@ -26,7 +26,7 @@
 
 @layout(
     pageTitle = titleNoForm(messages("addGuidance.title")),
-    showBackLink = false
+    showBackLink = true
```

Comment:
By Default is already true. so we can remove this everywhere unless needed explicitly false.

Reply:
Updated